### DISCUSSION TITLE
Offre évènements et affichage de l'établissement

### DISCUSSION THREAD

**Discussion by SandrineRobert** [unistra/esup-immersup-demandes#4] [2- Partie publique]
Serait-il possible de choisir de ne pas afficher l'établissement proposant des évènements ? Cela permettrait de supprimer un clic jusqu'à l'affichage des créneaux (voir exemple ci-dessous).
![Capture 01-12](https://github.com/unistra/esup-immersup-demandes/assets/152433302/1f597bb3-9f47-4412-9ef7-828fc4b83c3a)

**Comment by efinkbeiner5** [comment]
Vous n'avez qu'un seul établissement qui propose des immersions donc je comprends votre demandes mais il faut qu'on ait l'avis d'instance multiétablissement pour pouvoir prendre une décision.

**Comment by efinkbeiner5** [comment]
Solution envisagée : 
 - S'il y a qu'une seul item, l'ouvrir par défaut pour avoir un clic de moins. Est-ce qu'on doit le faire aussi au niveau du type d'évènement ? 
Chiffrage : faible

**Comment by efinkbeiner5** [comment]
Fait dans la version 4 en 2024
